### ISSUE TITLE
Order Creation: Accessibility → The popup's dismiss button should have a label

### ISSUE THREAD

**Issue by shiki** [woocommerce/woocommerce-ios#6066]
When using VoiceOver, this should at least say "Dismiss. Button" rather than just "Button". 

<img src="https://user-images.githubusercontent.com/198826/152424876-3aa52b1c-5c14-4623-8308-11a10beef0d2.png" width="300">j
